### PR TITLE
Attempt to fix OutOfMemoryError

### PR DESCRIPTION
**Pull request by labkey-jeckels**
#### Rationale
https://teamcity.labkey.org/buildConfiguration/LabkeyTrunk_DailySuites_DailyDPostgres/2014795?hideProblemsFromDependencies=false&hideTestsFromDependencies=false&expandBuildChangesSection=true&expandBuildProblemsSection=true&expandBuildTestsSection=true

#### Changes
* Fix code that merges old and new data

### CODE REVIEW

File: mothership/src/org/labkey/mothership/MothershipManager.java [hunk 1]
```diff
@@ -426,6 +426,7 @@ else if (StringUtils.isEmpty(currentValue))
         ObjectMapper mapper = new ObjectMapper();
         try
         {
+            log.info("Merging JSON. Old is " + currentValue + " characters, new is " + newValue.length());
```
Comment (labkey-tchad) [inline mothership/src/org/labkey/mothership/MothershipManager.java]: Should this be logged at debug level or are you just planning on removing it before merging?
```suggestion
            log.debug("Merging JSON. Old is " + currentValue + " characters, new is " + newValue.length());

```

**Review by labkey-tchad — APPROVED** [review]

File: mothership/src/org/labkey/mothership/MothershipManager.java (hunk 1 above)
Comment (labkey-adam) [reply mothership/src/org/labkey/mothership/MothershipManager.java]: Also, did you mean `currentValue.length()`?

**Review by labkey-adam — CHANGES_REQUESTED** [review]
Update logging

File: mothership/src/org/labkey/mothership/MothershipManager.java (hunk 1 above)
Comment (labkey-jeckels) [reply mothership/src/org/labkey/mothership/MothershipManager.java]: Yes

**Review by labkey-adam — APPROVED** [review]

**Pull request merged**